cardinal/server/server_test.go

```diff
@@ -146,10 +177,12 @@ func TestShutDownViaMethod(t *testing.T) {
 	assert.Assert(t, !w.IsGameLoopRunning())
 	_, err = http.Get("http://localhost:4040/health")
 	assert.Check(t, err != nil)
+	err = txh.Close()
+	assert.NilError(t, err)
 }
 
 func TestShutDownViaSignal(t *testing.T) {
-	setTestTimeout(t, 10*time.Second) // If this test is frozen then it failed to shut down, create a failure with panic.
+	//setTestTimeout(t, 10*time.Second) // If this test is frozen then it failed to shut down, create a failure with panic.
```

Comment:
do we need to uncomment this? if not lets just remove it

Reply:
Yes we do, that's suppose to be there, it's just removed for testing. 

If you look at the test not all tests are just asserts. Some tests are just spinning and a passing test is if it breaks out of those loops. A failing test would spin forever. The setTestTimeout places an assert on those spinlocks. Basically if a test spins for > 10s I classify it as an error and trigger a panic.